### PR TITLE
Display missions

### LINKED ISSUE #15: [4pt] Display missions - Lists render
- Use `useSelector()` Redux Hook to select the state slices and render lists of  missions in corresponding routes. i.e.:
```javascript
// get rockets data from the store
const rockets = useSelector(state => state.rockets);
```

- You can style the whole application "by hand" or you could use [React Bootstrap](https://react-bootstrap.github.io/), a UI library that could speed up the process. This is a popular library and working with its components would be good practice.
- Render a table with the missions' data (as per design).

### PR DESCRIPTION
**Pull request by notlfish**
### What?
Pull request to complete the rendering (some functionality is still missing) of the Missions page, following [the specifications](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/images/space-travelers__missions.png).

### Before merging
The following tasks are to be completed before merging":
- [x] Add a `MissionsList` component that retrieves the missions state from the store and displays a table of missions.
- [x] Add a `MissionEntry` component that displays the information for one mission in one row of the table.
- [x] Style the table to be similar to the specs.

### Note
To check how the table displays a mission that the user is a member of, make the following change in `src/utils/missions-api.js`:
Change line 16 from
```javascript
    status: 'Not a member',
```
to
```javascript
    status: 'Active member',
```

### CODE REVIEW

File: src/components/missions/MissionEntry.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import PropTypes from 'prop-types';
+import Button from 'react-bootstrap/Button';
+import Badge from 'react-bootstrap/Badge';
```
Comment (Hamzaoutdoors) [inline src/components/missions/MissionEntry.js:3]: to import components from react-bootstrap you can regroup them to save more line in code like that .i.e : 
```JS
import { Button, Badge } from 'react-bootstrap';
```

File: src/components/missions/MissionEntry.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  if (status === 'Not a member') {
+    return (
+      <Button
+        className="text-nowrap mx-3"
+        variant="outline-secondary"
+      >
+        Join Mission
+      </Button>
+    );
+  }
```
Comment (Hamzaoutdoors) [inline src/components/missions/MissionEntry.js:15]: [OPTIONAL]: here you can use the Conditional (ternary) operator for ES6 for best practice and to reduce your code : 
```JS
function joinMessage(status) {
  return (status === 'Not a member' && <Button
        className="text-nowrap mx-3"
        variant="outline-secondary"
      >
        Join Mission
      </Button>);
}
```
and after that use it inside your JSX code

**Review by Hamzaoutdoors — APPROVED** [review]
# APPROVED  🟢  

Hi @notlfish  ,

Good job so far! I must applaud you for your wonderful efforts.

## Optional suggestions
Consider the optional changes nice to have features and only work on them when you have satisfied the required changes

_Cheers and Happy coding!👏👏👏_

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

File: src/components/missions/MissionEntry.js (hunk 1 above)
Comment (notlfish) [reply src/components/missions/MissionEntry.js:3]: I didn't know you could do this, thanks for the tip!

File: src/components/missions/MissionEntry.js (hunk 2 above)
Comment (notlfish) [reply src/components/missions/MissionEntry.js:15]: I didn't like that piece of code either. I'll find a better way to do it.

**Pull request merged**